server/src/app.ts

```diff
@@ -40,16 +40,19 @@ export abstract class AppBase {
   /** Configure Express middleware. */
   private middleware(): void {
     this.express.use(logger('dev'));
-    this.express.use(bodyParser.json());
-    this.express.use(bodyParser.urlencoded({ extended: false }));
+    this.express.use(bodyParser.json({limit: '5mb'}));
+    this.express.use(bodyParser.urlencoded({ limit: '5mb', extended: false }));
```

Comment:
Maybe add a comment why this limit is needed?

Reply:
done